math/fibonacci.c

```diff
@@ -1,23 +1,40 @@
+/**
+ * @file fibonacci.c
```

Comment:
No need to enter the file name manually, best let Doxygen detect it automatically
```suggestion
 * @file 
```

Reply:
I am not aware of what Doxygen is.

Reply:
It's what we use to generate the docs from the code, the contribution guidelines have more details, or you can check out their project website [here](https://www.doxygen.nl/index.html)

Reply:
Thanks for the help. I would check it.